apps/payments/next/middleware.ts

```diff
@@ -0,0 +1,62 @@
+import { NextRequest, NextResponse } from 'next/server';
+
+export function middleware(request: NextRequest) {
```

Comment:
This follows the example provided by Next.js docs, in addition to the Stripe CSPs

https://nextjs.org/docs/app/building-your-application/configuring/content-security-policy